### PR TITLE
Add Audio component

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: src/components/Audio/Audio.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+class Audio extends Video {
+  /**
+   * Render an audio element
+   */
+  render() {
+    const {innerRef, fallback, children} = this.props;
+
+    const {
+      tagAttributes, // Attributes of this video element
+      sources        // <source> tags of this video element
```
Comment (asisayag2) [inline src/components/Audio/Audio.js:20]: @maoznir , what are the default sources?

**Comment by asisayag2** [comment]
We are waiting with this until we get a 'go' from @mickey-gordon

**Comment by mickey-gordon** [comment]
I gave the go at noon.

**Comment by asisayag2** [comment]
Great, @mickey-gordon.
Thanks!

File: src/components/Audio/Audio.js (hunk 1 above)
Comment (maoznir) [reply src/components/Audio/Audio.js:20]: good point, how about aac,mp3,wav?

Comment (maoznir) [reply src/components/Audio/Audio.js:22]: was set to aac,mp3,ogg

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "node_modules/.bin/mocha --require @babel/register test/.setup.js --recursive test",
-    "test:all": "run-s compile dist test test-dist test-lib",
+    "test:all": "run-s build test test-dist test-lib",
```
Comment (asisayag2) [inline package.json:8]: @maoznir  why is that needed as part of this PR?

File: docs/index.html [hunk 3]
```diff
@@ -11,4 +11,4 @@
     }
   } catch (e) {
     console.warn('unable to connect to parent frame for connecting dev tools');
-  }</script></head><body><div id="root"></div><script src="./sb_dll/storybook_ui_dll.js"></script><script src="runtime~main.3eb38892541d21003b89.bundle.js"></script><script src="vendors~main.b90e3eaaa88ee385794c.bundle.js"></script><script src="main.b9be9fea23636e16b87e.bundle.js"></script></body></html>
\ No newline at end of file
+  }</script></head><body><div id="root"></div><script src="./sb_dll/storybook_ui_dll.js"></script><script src="runtime~main.23a842e0cf9f5bd27e60.bundle.js"></script><script src="vendors~main.cf21e66639e7e7094b5b.bundle.js"></script><script src="main.8a2064d11d740364a590.bundle.js"></script></body></html>
```
Comment (asisayag2) [inline docs/index.html:14]: What are those?

File: src/components/Video/Video.js [hunk 4]
```diff
@@ -46,7 +48,8 @@ class Video extends CloudinaryComponent {
   generateSources = (cld, publicId, childTransformations, sourceTransformations, sourceTypes) => (
     sourceTypes.map(sourceType => {
       const src = this.generateVideoUrl(cld, publicId, childTransformations, sourceTransformations, sourceType);
-      const mimeType = 'video/' + (sourceType === 'ogv' ? 'ogg' : sourceType);
+      //const mimeType = 'video/' + (sourceType === 'ogv' ? 'ogg' : sourceType);
```
Comment (asisayag2) [inline src/components/Video/Video.js:51]: Please remove

File: test/AudioTest.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+chai.use(require('chai-string'));
+
+describe('Audio', () => {
+  it("should include child transformation for a single source type", function () {
+    let tag = shallow(
+      <Audio cloudName='demo'
+             sourceTypes={'webm'}
+             publicId='dog'
+             sourceTransformation={{
+               webm: {overlay: 'text:verdana_30:webm!'}
```
Comment (asisayag2) [inline test/AudioTest.js:16]: Kinda funny use-case... text overlay over an Audio tag. You can add a video layer

File: test/AudioTest.js [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+             sourceTypes={'webm'}
+             publicId='dog'>
+        <Transformation fps='25-'/>
+      </Audio>);
+    expect(tag.props().src).to.endWith('/fps_25-/dog.webm');
+  });
+
+  it('should support startOffset parameter', function () {
+    let tag = shallow(
+      <Audio cloudName="demo" sourceTypes={'webm'} publicId="dog">
```
Comment (asisayag2) [inline test/AudioTest.js:49]: @maoznir , please change the tests to be audio-oriented... I understand that the mechanism of setting the sourceType to webm or mp3 is identical, but still... it doesn't look good

File: package.json (hunk 2 above)
Comment (maoznir) [reply package.json:8]: @asisayag2 caught this when test:all failed, compile doesn't exist anymore.

File: src/components/Video/Video.js (hunk 4 above)
Comment (asisayag2) [reply src/components/Video/Video.js:51]: Comment is still there.
